src/index.html

```diff
@@ -2,27 +2,35 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link rel="stylesheet" href="index.css" />
-    <script
-      src="https://kit.fontawesome.com/bcb41d9ea1.js"
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
       crossorigin="anonymous"
-    ></script>
+    />
+    <title>To Do List</title>
+    <script defer src="main.bundle.js"></script>
   </head>
+
   <body>
-    <div class="container">
-      <header>Todo List</header>
-      <div class="inputField">
-        <input type="text" placeholder="Add to list..." />
-        <button class="add-btn"><i class="fas fa-level-down-alt"></i></button>
+    <section class="todo-list">
+      <div class="todo-list-header">
```

Comment:
Per SEO, accessibility, and as an HTML good practice, it would be better to wrapp all the primary content of the webpage in a ```main``` tag.